tests/components/tibber/test_services.py

```diff
@@ -110,96 +106,60 @@ def generate_mock_home_data():
     return mock_homes
 
 
-def create_mock_tibber_connection():
-    """Create a mock tibber connection."""
-    tibber_connection = MagicMock()
-    tibber_connection.get_homes.return_value = generate_mock_home_data()
-    return tibber_connection
-
-
-def create_mock_hass():
-    """Create a mock hass object."""
-    mock_hass = MagicMock
-    mock_hass.data = {"tibber": create_mock_tibber_connection()}
-    return mock_hass
-
-
+@pytest.mark.parametrize(
+    "data",
+    [
+        {},
+        {"start": STARTTIME.isoformat()},
+        {
+            "start": STARTTIME.isoformat(),
+            "end": (STARTTIME + dt.timedelta(days=1)).isoformat(),
+        },
+    ],
+)
 async def test_get_prices(
+    mock_tibber_setup: MagicMock,
+    hass: HomeAssistant,
     freezer: FrozenDateTimeFactory,
+    data,
 ) -> None:
     """Test __get_prices with mock data."""
```

Comment:
Please update all docstrings.
```suggestion
    """Test get_prices with mock data."""
```